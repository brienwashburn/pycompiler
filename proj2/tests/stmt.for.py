for a in b: pass
else: bar

for a, b, c in q, r, x:
    pass

for a, *b, *c, in x, y, z,:
    a = x
    b = y
    z = z
else:
    a = y
