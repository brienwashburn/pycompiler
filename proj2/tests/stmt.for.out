(Module
 (For
  (target (Name a))
  (iter (Name b))
  (body (Pass))
  (orelse (Expr (Name bar))))
 (For
  (target (Tuple (Name a) (Name b) (Name c)))
  (iter (Tuple (Name q) (Name r) (Name x)))
  (body (Pass))
  (orelse))
 (For
  (target (Tuple (Name a) (Starred (Name b)) (Starred (Name c))))
  (iter (Tuple (Name x) (Name y) (Name z)))
  (body
   (Assign (targets (Name a)) (value (Name x)))
   (Assign (targets (Name b)) (value (Name y)))
   (Assign (targets (Name z)) (value (Name z))))
  (orelse (Assign (targets (Name a)) (value (Name y))))))
